### PR TITLE
Fix PHPCS issues and Remove Google+ Counter Functionality.

### PR DESCRIPTION
**Pull request by wpamitkumar**
-Fix PHPCS issues.
-Compatible with WordPress 5.1 
-Remove Google+ counter because of [Google+ API shutdown](https://developers.google.com/+/api-shutdown)

### CODE REVIEW

File: source.php [hunk 1]
```diff
@@ -17,1361 +31,1122 @@
...
+
+			if ( 'recommend_light' === $options['fb_style']
+			|| 'recommend_dark' === $options['fb_style'] ) {
+				$rt_social_text = 'Recommend';
+			} else {
+				$rt_social_text = 'Share';
+			}
+		}
+
+		if ( 'horizontal' === $options['display_options
```
Comment (dishitpala) [inline source.php]: There is no use of this wrapper. Use `filemtime` directly

File: source.php [hunk 2]
```diff
@@ -760,20 +646,17 @@ function rtsocial( $args = array() ) {
...
-	&& ! empty( $options['google_api_key'] ) ) {
+	&& in_array( 'gplus', $options['active'], true ) ) {
 		$gplus = array_search( 'gplus', $options['active'], true );
 
-		$gplus_count = ( empty( $options['hide_count'] ) || 1 !== (int) $options['hide_count'] ) ? '<div class="rtsocial-' . $options['display_options_set'] . '-count"><div class="rtsocial-' . $options['display_options_set'] . '-notch"></div><span class="rtsocial-gplus-count"></span></div>' : '';
-
 		$gplus_layout = '<div class="rtsocial-gplus-' . $options['display_options_set'] . '">';
 
 		if ( 'horizontal' === $options['display_options_set'] ) {
-			$gplus_layout .= '<div class="rtsocial-gplus-' . $options['display_options_set'] . '-button"><a class="rtsocial-gplus-button" href= "https://plus.google.com/share?url=' . $rtslink . '" rel="nofollow" target="_blank" title="+1: ' . $rtatitle . '"></a></div>' . $gplus_count;
```
Comment (dishitpala) [inline source.php]: Use `sprintf` instead of concatenation.
It will be easier to find the things right.

File: source.php [hunk 3]
```diff
@@ -760,20 +646,17 @@ function rtsocial( $args = array() ) {
...
+			$gplus_layout .= '<div class="rtsocial-gplus-' . $options['display_options_set'] . '-button"><a class="rtsocial-gplus-button" href= "https://plus.google.com/share?url=' . $rtslink . '" rel="nofollow" target="_blank" title="+1: ' . $rtatitle . '"></a></div>';
 		} else {
 
 			if ( 'vertical' === $options['display_options_set'] ) {
-				$gplus_layout .= $gplus_count . '<div class="rtsocial-gplus-' . $options['display_options_set'] . '-button"><a class="rtsocial-gplus-button" href= "https://plus.google.com/share?url=' . $rtslink . '" rel="nofollow" target="_blank" title="+1: ' . $rtatitle . '"></a></div>';
+				$gplus_layout .= '<div class="rtsocial-gplus-' . $options['display_options_set'] . '-button"><a class="rtsocial-gplus-button" href= "https://plus.google.com/share?url=' . $rtslink . '" rel="nofollow" target="_blank" title="+1: ' . $rtatitle . '"></a></div>';
```
Comment (dishitpala) [inline source.php]: Use sprintf instead of concatenation.
It will be easier to find the things right.

**Comment by dishitpala** [comment]
Block Comments Should be `/** */` not `/* */`
Use `sprintf` and `printf` instead of concatenation of string

File: source.php (hunk 3 above)
Comment (coderboy007) [reply source.php]: Used sprintf() instead of concatenation for HTML in the whole plugin.

File: source.php (hunk 1 above)
Comment (coderboy007) [reply source.php]: used `filemtime` directly.

**Pull request merged**